tf2_web_republisher/AUTHORS.md

```diff
@@ -0,0 +1,11 @@
+Original Authors
+----------------
+
+ * Julius Kammerl (jkammerl@willowgarage.com)
+ * Paul Gesel (paul.gesel@picknik.ai)
```

Comment:
While it is a full rewrite, technically you're not an original author.

Maybe you can change this to just "authors" since you should get the credit for the ROS 2 port!

Reply:
How about now?